feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -450,37 +389,37 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []string
 		}
 
 		if p.idValidation {
-			id := gnmi.Get(t, dut, component.Id().State())
-			t.Logf("Hardware card %s Id: %s", card, id)
-			if id == "" {
-				t.Errorf("component.Id().Get(t) for %q): got empty string, want non-empty string", card)
+			if !(*deviations.SwitchChipIDUnsupported && p.pType == oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_INTEGRATED_CIRCUIT) {
+				id := gnmi.Get(t, dut, component.Id().State())
+				t.Logf("Hardware card %s Id: %s", card, id)
+				if id == "" {
+					t.Errorf("component.Id().Get(t) for %q): got empty string, want non-empty string", card)
+				}
 			}
 		}
-
 		if p.nameValidation {
 			name := gnmi.Get(t, dut, component.Name().State())
 			t.Logf("Hardware card %s Name: %s", card, name)
 			if name == "" {
 				t.Errorf("component.Name().Get(t) for %q): got empty string, want non-empty string", card)
 			}
 		}
-
 		if p.partNoValidation {
-			partNo := gnmi.Get(t, dut, component.PartNo().State())
-			t.Logf("Hardware card %s PartNo: %s", card, partNo)
-			if partNo == "" {
-				t.Errorf("component.PartNo().Get(t) for %q): got empty string, want non-empty string", card)
+			if !(*deviations.SwitchChipPartNoUnsupported && p.pType == oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_INTEGRATED_CIRCUIT) {
```

Comment:
i have discussed this with google team and we may drop this check without a deviation

Reply:
@vosipchu is there a Google bug capturing this conversation?

Reply:
LGTM for removal